### PR TITLE
Run2-alca114 Update the macros

### PR DESCRIPTION
**Pull request by bsunanda**
The modified macros are used to calibrate HCAL  using Isotracks and also to validate new Geant4 releases using collision data

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
The test failure has nothing to do with this PR. There was time out in one test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibMonitor.C [hunk 1]
```diff
@@ -698,8 +698,8 @@ void CalibMonitor::Loop() {
     Long64_t ientry = LoadTree(jentry);
     if (ientry < 0) break;
     nb = fChain->GetEntry(jentry);   nbytes += nb;
-    if (jentry%10000 == 0) std::cout << "Entry " << jentry << " Run " << t_Run
-				     << " Event " << t_Event << std::endl;
+    if (jentry%100000 == 0) std::cout << "Entry " << jentry << " Run " << t_Run
```
Comment (ghellwig) [inline Calibration/HcalCalibAlgos/macros/CalibMonitor.C]: braces are missing here, too

**Review by ghellwig — COMMENTED** [review]
thanks for taking care of the comments so far

File: Calibration/HcalCalibAlgos/macros/CalibMonitor.C (hunk 1 above)
Comment (ghellwig) [reply Calibration/HcalCalibAlgos/macros/CalibMonitor.C]: ignore this comment; since it is all on one line, it is fine for me

**Comment by ghellwig** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**